installer/manifests/keptn/core.yaml

```diff
@@ -1,32 +1,230 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: api-nginx-config
+  namespace: keptn
+data:
+  nginx.conf: |
+    user nginx;
+    worker_processes  3;
+    error_log /dev/stdout info;
+    events {
+      worker_connections  10240;
+    }
+
+    http {
+      log_format  main
+              'remote_addr:$remote_addr\t'
+              'time_local:$time_local\t'
+              'method:$request_method\t'
+              'uri:$request_uri\t'
+              'host:$host\t'
+              'status:$status\t'
+              'bytes_sent:$body_bytes_sent\t'
+              'referer:$http_referer\t'
+              'useragent:$http_user_agent\t'
+              'forwardedfor:$http_x_forwarded_for\t'
+              'request_time:$request_time';
+      access_log /dev/stdout;
+      server {
+          listen       80;
+          server_name  _;
+
+          location /configuration-service/swagger-ui/swagger.yaml {
+            # auth via backend (if the subrequest returns a 2xx response code, the access is allowed. If it returns 401 or 403,
+            # the access is denied) before we store the file
+            # see http://nginx.org/en/docs/http/ngx_http_auth_request_module.html
+
+            rewrite /configuration-service/(.*) /$1  break;
+            proxy_pass         http://configuration-service:8080;
+            proxy_redirect     off;
+            proxy_set_header   Host $host;
+            proxy_http_version 1.1;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
+          }
+
+          location  /configuration-service {
+            # auth via backend (if the subrequest returns a 2xx response code, the access is allowed. If it returns 401 or 403,
+            # the access is denied) before we store the file
+            # see http://nginx.org/en/docs/http/ngx_http_auth_request_module.html
+            auth_request               /v1/auth;
+
+            rewrite /configuration-service/(.*) /$1  break;
+            proxy_pass         http://configuration-service:8080;
+            proxy_redirect     off;
+            proxy_set_header   Host $host;
+            proxy_http_version 1.1;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
+          }
+
+          location ~* /v1/project/[a-z0-9-]+/resource {
```

Comment:
we should document that this is a legacy endpoint for v1, that will be removed in the future in favour of directly using configuration-service.

Reply:
Done.